en/tensor-examples.html

```diff
@@ -0,0 +1,246 @@
+---
+# Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+title: "Tensor Computation Examples"
+---
+
+<p>It is well know that tensors can be used to express machine-learned models such as neural nets,
+but they can be used for much more than that. The tensor model in Vespa is especially powerful,
+since it supports sparse dimensions, dimension names and lambda computations. Whatever you want
```

Comment:
per @geirst we should use "mapped and indexed" dimensions as names (i.e. not sparse). However, here, there property of the tensor is being sparse (a mapped tensor is both sparse and uses names), so OK.

Reply:
Some reference: https://docs.vespa.ai/en/reference/tensor.html. Here we lay the foundation where a tensor dimension is either mapped or indexed. The tensor as a whole can be sparse (only mapped dimensions), dense (only indexed dimensions), mixed (a combination).